docs/user/Hootenanny-id.asciidoc

```diff
@@ -224,14 +224,50 @@ Once a bounding box (or visual extent) has been selected, additional options to
 
 [[Hoot_Clip_Map]]
 .Clipping a layer using a defined bounding box. The imagery basemap displayed is the DigitalGlobe GBM service.
-image::user/images/id/hoot_clip.png[scaledwidth="75%"]
+image::images/id/hoot_clip.png[scaledwidth="75%"]
+
+===== Grail Tools
+
+[[Hoot_grailTools_Map]]
+.Grail Pull and Derive Differential Changeset found under the tool dropdown.
+image::images/id/GrailTools.png[scaledwidth="75%"]
+
+*Grail pull from OSM Overpass and the Rails Port*
+
+Under the tools section in the Hootenanny UI there will be a section labeled "Grail Tools". Under that
+you should see the option to "Pull Remote Data". Once selected you will be prompted to select a bounding
+box region by either drawing the bounding box or using the visual extent as your bounding box. Once selected
+you can submit the job for running. After the job has completed you should see a folder with the name of
+grail_*coordinates for your bounding box* which should contain a filed named "reference" (the Rails Port data)
+and a file named "secondary" (the OSM Overpass data).
+
```

Comment:
Mention how you can use the reference and secondary layers to run any kind of conflation, step through reviews in the merged layer, then derive a changeset and upload it to remote rails port